kasa/protocol.py

```diff
@@ -361,6 +363,18 @@ def decrypt(ciphertext: bytes) -> str:
         ).decode()
 
 
+def get_default_credentials(tuple: Tuple[str, str]) -> Credentials:
+    """Return decoded default credentials."""
+    un = base64.b64decode(tuple[0].encode()).decode()
+    pw = base64.b64decode(tuple[1].encode()).decode()
+    return Credentials(un, pw)
+
+
+DEFAULT_CREDENTIALS = {
+    "KASA": ("a2FzYUB0cC1saW5rLm5ldA==", "a2FzYVNldHVw"),
+    "TAPO": ("dGVzdEB0cC1saW5rLm5ldA==", "dGVzdA=="),
+}
```

Comment:
Should we rather contain these inside the transport classes themselves?

Reply:
They're not transport specific, i.e. the TAPO one will work with klap and aes transport with SmartProtocol.  Also there's a chance that the KASA one works with SmartProtocol KASA devices although we don't know for sure.